enterprise/frontend/src/embedding-sdk/cli/snippets/analytics-page-snippet.ts

```diff
@@ -3,16 +3,17 @@
  * theme switcher, and a sample dashboard.
  */
 export const ANALYTICS_PAGE_SNIPPET = `
+import { AnalyticsProvider } from './analytics-provider'
+import { EmbeddingProvider } from './embedding-provider'
 import { AnalyticsDashboard } from './analytics-dashboard'
-import { MetabaseEmbedProvider, SampleThemeProvider } from './metabase-provider'
 
 import './analytics.css'
 
 export const AnalyticsPage = () => (
-  <SampleThemeProvider>
-    <MetabaseEmbedProvider>
+  <AnalyticsProvider>
+    <EmbeddingProvider>
       <AnalyticsDashboard />
```

Comment:
AnalyticsProvider is for the demo components to function (e.g. theme switching, user switching), while the EmbeddingProvider configures `MetabaseProvider`.